src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/CSharpTagHelperCodeRenderer.cs

```diff
@@ -456,6 +457,21 @@ private void RenderTagOutput(string tagOutputMethodName)
                    .WriteEndMethodInvocation();
         }
 
+        private void RenderSelfClosing(bool selfClosing)
+        {
+            if (_designTimeMode)
+            {
+                return;
+            }
+
+            _writer.Write(ExecutionContextVariableName)
+                   .Write(".")
+                   .Write(_tagHelperContext.ExecutionContextSelfClosingPropertyName)
+                   .Write(" = ")
```

Comment:
I copied that codes from a different method in the same class I'll update the original one too when I do the actual PR. :-)